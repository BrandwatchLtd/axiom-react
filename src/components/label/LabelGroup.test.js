import React from 'react';
import renderer from 'react-test-renderer';
import { LabelGroup, Label } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <LabelGroup { ...props }>
      <Label>Lorem ipsum</Label>
    </LabelGroup>
  );
}

describe('LabelGroup', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
