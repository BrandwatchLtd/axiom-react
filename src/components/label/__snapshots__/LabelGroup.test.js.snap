exports[`LabelGroup renders with defaultProps 1`] = `
<div
  className="ax-label__group">
  <span
    className="ax-label ax-label--white ax-label--medium">
    Lorem ipsum
  </span>
</div>
`;
